flutter_vlc_player/lib/src/vlc_player_value.dart

```diff
@@ -114,8 +118,8 @@ class VlcPlayerValue {
 
   /// A description of the error if present.
   ///
-  /// If [hasError] is false this is [null].
-  final String? errorDescription;
+  /// If [hasError] is false this is [empty].
+  final String errorDescription;
```

Comment:
we can't use `null` anymore because we want to update the `errorDescription` in vlc controller and the only way to do this is through `value.copyWith`.
in this method because of null-safety we never can set the value of variable to null (the current implementation is `errorDescription: errorDescription ?? this.errorDescription,`).

i updated the mentioned comment to `/// If [hasError] is false this is [VlcPlayerValue.noError].` to resolve that confusion.